src/main/java/org/jenkinsci/test/acceptance/po/Login.java

```diff
@@ -1,6 +1,7 @@
 package org.jenkinsci.test.acceptance.po;
 
 import org.junit.Assert;
+import org.openqa.selenium.By;
```

Comment:
Not needed, the already present `by` object has a `name` method, see [ByFactory#L113](https://github.com/jenkinsci/acceptance-test-harness/blob/master/src/main/java/org/jenkinsci/test/acceptance/ByFactory.java#L113

Reply:
Good catch